services/core-web/src/components/Forms/permits/conditions/Condition.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Col, Row, Popconfirm, Button } from "antd";
+import SubCondition from "@/components/Forms/permits/conditions/SubCondition";
+import Section from "@/components/Forms/permits/conditions/Section";
+import ListItem from "@/components/Forms/permits/conditions/ListItem";
+
+const propTypes = {
+    condition: PropTypes.objectOf(PropTypes.any)
+};
+
+const defaultProps = {
+};
+
```

Comment:
children should be in defaultProps if not required

Reply:
Added isRequired